docs/dictionary/constant/nothing.lcdoc

```diff
@@ -0,0 +1,19 @@
+Name: nothing
+
+Type: constant
+
+Syntax: nothing
+
+Summary:
+The value with no type
+
+Example:
+put nothing is strictly nothing -- true
+
+Description:
+The constant <nothing> is the only value with no type, thus
```

Comment:
The constant <nothing> is a unique value which is used to represent the notion of no value. It converts to empty when used in all operators, but can be distinguished from empty using the <is strictly nothing> operator.